desktop-exporter/server.go

```diff
@@ -68,3 +110,23 @@ func (s Server) Start() error {
 func (s Server) Close() error {
 	return s.server.Close()
 }
+
+func getTraceDuration(spans []SpanData) (int64, error) {
+	if len(spans) < 1 {
+		return 0, errors.New("can't calculate trace duration - spans slice is empty")
+	}
```

Comment:
Consider returning a `time.Duration` object and only convert it to `duration.Milliseconds()` in your handler